addons/adminMenu/functions/fnc_uihook_teleportButton.sqf

```diff
@@ -40,6 +40,13 @@ private _return = if (_vehicle == _leader) then {
         ["Teleporting to position", "No empty position for teleport"] select _success
     }
 } else {
+    [{ // hint if TP-to-Vic fails
```

Comment:
(nitpick)
niche case, but if someone is fucking around in the start zone and enter a vehicle they won't have this pop

or if we add a shitty addon that for some reason makes this return an object